libimage/pull.go

```diff
@@ -351,7 +351,7 @@ func (r *Runtime) copySingleImageFromRegistry(ctx context.Context, imageName str
 	// attempt pulling that instead of doing the full short-name dance.
 	localImage, resolvedImageName, err = r.LookupImage(imageName, nil)
 	if err != nil && errors.Cause(err) != storage.ErrImageUnknown {
-		return nil, errors.Wrap(err, "error looking up local image")
+		logrus.Errorf("Error looking up %s in local storage: %v", imageName, err)
```

Comment:
The information that we encounter an error is already part of the verbosity level, so I would suggest removing the word "Error":
```suggestion
		logrus.Errorf("Looking up %s in local storage: %v", imageName, err)
```

Reply:
SGTM